### ISSUE TITLE
Upgrade Guide needs to be updated with 0.x.x -> 1.0.0

### ISSUE THREAD

**Issue by mateusbandeiraa** [axios/axios#5014]
#### Section/Content To Improve
Upgrade guide

#### Suggested Improvement
Axios v1.0.0 has breaking changes (by looking at both the Issues page and my own application). The Upgrade Guide should be updated to include instructions on how to better migrate to the new major version.

#### Relevant File(s)
[UPGRADE_GUIDE.md](https://github.com/axios/axios/blob/v1.x/UPGRADE_GUIDE.md)

**Comment by MohamadKh75** [comment]
Where can i see the list of breaking changes?

**Comment by ghiscoding** [comment]
I asked the same question in a Discussion #4996 and the maintainer's answer was

> I will be adding one as soon as possible.

**Comment by LucaRed** [comment]
Any news on this?

**Comment by erik-itland** [comment]
What is going on here? So far UPGRADE_GUIDE.md has seems to have been removed instead of updated, and even after looking twice on the release notes it is absolutely not obvious how to solve the error messages I see.

**Comment by erik-itland** [comment]
Yes, but that is empty as well : )

**Comment by TheMoonDawg** [comment]
Any status on this @jasonsaayman ? Looks like the migration guide is still empty. :)

**Comment by bmuenzenmeyer** [comment]
I've quick created https://github.com/bmuenzenmeyer/axios-1.0.0-migration-guide/

- we as a community of 99k stars and 40+ million weekly downloads have the collective wisdom to track these changes, spread the word, add what you can! I'll be encouraging my team to do so.
- (no warranties, but I hope it helps some folks until something official lands)

**Comment by pieterocp** [comment]
@bmuenzenmeyer do you want to make a PR which is just an update to the Upgrade Guide linking to that repo? Used it for a project and it worked well enough. Since it's been over a year and anyone doing an upgrade is greeted with an empty document which.

**Comment by bmuenzenmeyer** [comment]
@pieterocp I am glad that you found it useful! Agree empty guides are concerning...but my intention is to retain it's community presence until official guidance surfaces. I don't want a PR to sit idle and unwanted. People seeking upgrade guidance are finding this thread and others which point to my guide.

**Comment by imtushardave** [comment]
@jasonsaayman  Cross-Site Request Forgery Vulnerability is found in older versions of Axios and we are still waiting for the migration guide. Would you happen to have any updates on this? or at least let us know how much time it will take.

**Comment by bmuenzenmeyer** [comment]
> Cross-Site Request Forgery Vulnerability is found in older versions of Axios

this PR https://github.com/axios/axios/pull/6091 also fixes it

**Comment by VanessaHenderson** [comment]
@jasonsaayman I understand you're a busy human but is there any updates on an official migration guide?

**Comment by KrisLau** [comment]
@jasonsaayman Hi just checking in, is the migration guide still in the plans or has that been scrapped in favor of waiting until a fully stable release (if that is not yet been reached #5114)?

**Comment by doutatsu** [comment]
How is this library on version 1.7 and we still don't have a guide to upgrade to 1.7 😂 I am still on a years old version because of that, it's pretty crazy to me

**Comment by mabasic** [comment]
> How is this library on version 1.7 and we still don't have a guide to upgrade to 1.0 😂 I am still on a years old version because of that, it's pretty crazy to me

I was in the same boat, the solution is to just upgrade and yolo, or switch to fetch

**Comment by doutatsu** [comment]
I am looking to migrate to [Ky](https://github.com/sindresorhus/ky), not a fan of fetch

**Comment by KrisLau** [comment]
@DigitalBrainJS Any news on the migration guide? Sorry for the sudden tag but I noticed you were one of the main contributors for the recent releases

**Comment by jasonsaayman** [comment]
im adding it to the new docs, we will probably publish in about 2 weeks it includes everything added here and some more

**Comment by KrisLau** [comment]
@jasonsaayman Did it get added?

**Comment by pieterocp** [comment]
> [@jasonsaayman](https://github.com/jasonsaayman) Did it get added?

I don't think this has been added yet? From the documentation, in the notes section it links back to the main branch's `UPGRADE_GUIDE`, this only details between versions of the 0.X series. And the migration guide only has a heading for the time being?

https://github.com/axios/axios/blob/main/UPGRADE_GUIDE.md

https://github.com/axios/axios/blob/v1.x/MIGRATION_GUIDE.md

Tbh, used the community guide - https://github.com/axios/axios/issues/5014#issuecomment-1498357937 - to do the upgrade which worked well enough.

**Comment by KrisLau** [comment]
@jasonsaayman  ?

**Comment by ekilah** [comment]
~Following up / bumping as another interested party.~

Actually, as of 

- https://github.com/axios/axios/pull/7218

it appears that

https://github.com/axios/axios/blob/v1.x/MIGRATION_GUIDE.md

exists and has content.

Should this now be closed?

**Comment by jasonsaayman** [comment]
We have provided this in the docs. The new site has a guide and AI should easily be able to assist this process.
